samples-aspnetcore-2x/social-login/okta-aspnetcore-mvc-example/Views/Account/SignIn.cshtml

```diff
@@ -22,15 +22,15 @@
         customButtons: [{
                 title: 'Sign in with Facebook',
                 className: 'social-auth-button social-auth-facebook-button link-button',
-                click: function() {
-                    window.location.href = 'https://localhost:44314/Account/SignInWithIdp?idp=@Configuration["okta:FacebookIdp"]';
+                click: function () {
+                    window.location = new URL('/Account/SignInWithIdp?idp=@Configuration["okta:FacebookIdp"]', window.location);
```

Comment:
should this be `window.location.href = .. ` same as below?

Reply:
I thought `windows.location.href = smth` doesn't work for partial URLS, but it does, just checked it. I will update this code soon.

Reply:
If `window.location` works, we can use it too. Since I saw you are using both (with `href` and without `href`) I wasn't sure which one you wanted to use here.

Reply:
Yes, this works, but I changed these lines in order to make changes simpler. 
@laura-rodriguez There are 5 commits here now, could you squash them on merge? I guess this option is available to you.

Reply:
Sure, I'll do a squash merge.